addons/mail/models/mail_thread.py

```diff
@@ -1250,6 +1250,14 @@ def _message_parse_extract_payload_postprocess(self, message, payload_dict):
             body = etree.tostring(root, pretty_print=False, encoding='unicode')
         return {'body': body, 'attachments': attachments}
 
+    def _part_get_content_str(self, part):
```

Comment:
Don't like the "_str" suffix, purpose is more being defensive than being a string. If get_content does not raise we don't do anything more... Maybe ``def _part_get_content_decoded`` ? Not sure it is better, just trying out ...

Reply:
it is better